vcr/vcr.go

```diff
@@ -643,7 +653,7 @@ func (c *vcr) convert(query concept.Query) map[string]leia.Query {
 	return qs
 }
 
-func (c *vcr) generateProof(credential *vc.VerifiableCredential, kid ssi.URI) error {
+func (c *vcr) generateProof(credential *vc.VerifiableCredential, kid ssi.URI, key crypto.KeySelector) error {
```

Comment:
`KeySelector` has `KID` property which can be used instead of passing `kid`?

Reply:
different types: `ssi.URI` vs `string`